### ISSUE TITLE
Improvements to `helpdm` and a feature request for help channel notifications.

### ISSUE THREAD

**Issue by Qwerty-133** [python-discord/bot#1960]
This issue consists of two parts:

1. I believe we can improve `helpdm`'s embeds by making changes such as:
   - Adding a mention to the claimant
   - Setting their profile picture as the thumbnail
   - Adding the condensed text that the claimant themselves receive
   - Updating the embed when the help channel has been closed

2. While `helpdm` sometimes comes in handy when juggling between multiple help channels, what I would like more is to have the option to be notified for messages that I might have missed from the claimant. In terms of implementation, getting a DM from the bot if I haven't sent a message in the help channel 5 minutes after the claimant's message is what I had in mind.

**Comment by wookie184** [comment]
I agree with all points made, and think they would make the helpdm system much more useful.

Were you intereseted in implementing any of this @Qwerty-133? (regardless, as a note if anyone would like to implement just part of this that would be fine as there's a couple of different things here)

**Comment by Qwerty-133** [comment]
I'll start working on `helpdm` for now.

**Comment by minalike** [comment]
With our migration to forum channels this week we are now using threads (posts) instead of channels, and closed/dormant channels now get referred to as `#deleted-channel`.

I'd like to suggest we add the channel name as text in the embed somewhere as well.
